src/devices/acurite.c

```diff
@@ -595,8 +597,10 @@ static int acurite_986_callback(bitbuffer_t *bitbuf) {
 	}
 
 	if ((status & 1) == 1) {
+            if (debug_output) {//All fprintf()'s must be under debug output guards.
 	    fprintf(stderr, "%s Acurite 986 sensor 0x%04x - %d%c: low battery, status %02x\n",
 		    time_str, sensor_id, sensor_num, sensor_type, status);
+            }
```

Comment:
Ok, same question, why is some PR about sample code, going to change the behavior of a currently working device?

I don't think there should be any changes to acurite.c in this PR.   Ideally if the behavior of a device is getting changed it should be pretty explicit.  (especially there hasn't been any releases or tags to make it easier to go back to a known working version.)